docs/quick_start/installing.rst

```diff
@@ -270,5 +263,5 @@ Here, The GUI is forwarded to port 8082 on the host machine, it can be any port
     docker exec -it fledge bash
 
 .. note::
-    For Ubuntu 18.04 setup, you just need to replace ubuntu2004 with ubuntu1804.
-    Images are currently only available for Ubuntu version 18.04 and 20.04.
+    To set up Ubuntu 18.04, simply replace ubuntu2004 with ubuntu1804.
+    Currently, images are only available for Ubuntu versions 18.04 and 20.04, and are compatible with both Intel and AMD architectures.
```

Comment:
Ubuntu 22?

Reply:
Currently, we have not released the Docker image, which is why I omitted it from the documentation.